### DISCUSSION TITLE
Installed via homebrew for linux, where is the config json file?

### DISCUSSION THREAD

**Discussion by antonispgs** [fastfetch-cli/fastfetch#1294] [Q&A] [answered]
Can't find it in .config or in the .homebrew folder, unless I am missing something obvious. I would appreciate the help!

**Comment by awlobo** [comment, marked as answer]
You would need to run `fastfetch --gen-config`. It is in the usage section: https://github.com/fastfetch-cli/fastfetch#usage

**Reply by antonispgs** [reply]
Amazing, thanks do you happen to know if there is a way to make keys bigger in size?
They are too small in comparison to the text next to them.
![image](https://github.com/user-attachments/assets/d8b3c1f2-a849-49ab-a204-b40c11c5b366)

**Reply by awlobo** [reply]
Related to keys there are:

```
--key-width <num>                      Align the width of keys to <num> characters
--key-padding-left <num>               Set the left padding of keys to <num> characters
```

You can try, but I don't think is going to affect the size itself. 
Maybe try to open a new discussion so the question does not get lost here because I'm curious as well.

**Reply by CarterLi** [reply]
https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/JetBrainsMono#tldr

TLDR: use a nerd font without `mono`

**Reply by antonispgs** [reply]
> https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/JetBrainsMono#tldr
> 
> TLDR: use a nerd font without `mono`

That's it! Meslo LGS Nerd Regular is excellent for that!
